manifest_template.yml

```diff
@@ -46,25 +46,56 @@ apigee:
   - name: {{ ENV.name }}
     products:
       - name: {{ NAME }}
+        displayName: {{ TITLE }}
```

Comment:
display name should include the environment, it's really hard to tell these apart otherwise ..
e,g, 
displayName: Immunisation History - {{ MODE.displayName }} ({{ ENV.display_name }})
(unless TITLE includes this already ? )

Reply:
our title is:
{% set TITLE = PRODUCT_DISPLAY_NAME + '(' + ENV.display_name + ' Environment)' %}

Reply:
ah cool, I didn't spot that